scipy/stats/_multivariate.py

```diff
@@ -3521,3 +3522,353 @@ def rvs(self, eigs, random_state=None, tol=1e-13, diag_tol=1e-7):
+    >>> z = normal_invgamma.pdf(x, sig2, loc=2,
+                                    variance_scale=3, shape=5, scale=2); z
+    array([  5.15655226e-06,   3.07181064e-06,   1.87273568e-06,
+         1.16664399e-06,   7.41424234e-07,   4.79916138e-07,
+         3.15923431e-07,   2.11211796e-07,   1.43229349e-07,
+         9.84088858e-08])
+    >>> fig = plt.figure()
+    >>> ax = fig.add_subplot(111, projection='3d')
+    >>> ax.plot_surface(x, sig2, z)
+
+    """   
+    def __init__(self, seed=None):
+        super(normal_invgamma_gen, self).__init__(seed)
+        self.__doc__ = doccer.docformat(self.__doc__, nig_docdict_params)
+
+    def __call__(self, loc=0, variance_scale=1, shape=1, scale=1, seed=None):
+        """
+        Create a frozen normal inverse gamma distribution.
+
+        See `normal_invgamma_frozen` for more information.
+
+        """
+        return normal_invgamma_frozen(loc, variance_scale, shape, scale,
+                                    seed=seed)
+
+    def _check_parameters(self, loc, variance_scale, shape, scale):
+        if not np.isscalar(loc):
+            raise ValueError("""loc must be a scalar""")
+        if not np.isscalar(variance_scale) or variance_scale < 0:
+            raise ValueError("""variance_scale must be a positive scalar""")
+        if not np.isscalar(shape) or shape < 0:
+            raise ValueError("""shape must be a positive scalar""")
+        if not np.isscalar(scale) or scale < 0:
+            raise ValueError("""scale must be a positive scalar""")
+        return loc, variance_scale, shape, scale
+
+    def _check_input(self, x, sig2):
+        if x.ndim != 1:
+            raise ValueError("""array must be one dimensional""")
+        if sig2.ndim != 1:
+            raise ValueError("""array must be one dimensional""")
+        if any(sig2 < 0):
+            raise ValueError("""array must consist of positive values,
+                                as it represents variance""")
+        return x, sig2
+
+    def logpdf(self, x, sig2, loc=0, variance_scale=1, shape=1, scale=1):
+        """
+        Log of the Normal inverse gamma probability density function.
+
+        Parameters
+        ----------
+        x : array
+            One-dimensional array.
+        sig2 : array
+            One-dimensional array.
+        %(_nig_doc_default_callparams)s
+
+        Returns
+        -------
+        pdf : ndarray
+            Log of the probability density function evaluated at `(x, sig2)`.
+
+        """
+        loc, variance_scale, shape, scale = self._check_parameters(
+                                                    loc, variance_scale, shape, scale)
+        x, sig2 = self._check_input(x, sig2)
+
+        Zinv = shape * np.log(scale) - gammaln(shape) - 0.5 * (np.log(variance_scale) + _LOG_2PI)
+        out = Zinv - 0.5 * np.log(sig2) - (shape + 1.) * np.log(sig2) -\
+                scale/sig2 - 0.5/(sig2 * variance_scale) * (x-loc)**2
```

Comment:
Is there a mistake here? I think it should be 
`scale/sig2 - 0.5/sig2 * variance_scale * (x-loc)**2`
(Removed brackets)